Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# New Eras

## Changelog
- Added new era (Early Modern Era)
  - Military
    - Conscripts
    - High Velocity Bombs
    - Defense Charters
    - Riflemen (new unit in #380 )
    - Recruitment Centers
    - Professional Army
  - Civic
    - Construction Methods
    - Expansionism
    - Governance
    - High Eminence
    - Trams
    - Production Lines
- History Rework (#504)
  - Based off of: https://docs.google.com/spreadsheets/d/18zS-Ate6nsejx7rpzSmVuioRXps3iWSvSkx_xRJSWI/edit#gid=165545728
  - Reworked History of every culture
  - This is done to better fit the new era
- Riflemen unit (#380)
  - Still under the archer type unit and similar to the crossbowmen unit, the Riflemen will be a universally unlockable MAA unit once the player reaches Early Modern Era

  - Added: Riflemen unit
## TO DO :
  - [x] Fix images showing up as default, problem with era frames.
## How To Test
- Check all tech ideas look okay
- Check all images are okay
- Check all descriptions are okay
- Check history is okay
- Check Riflemen unit works
